shapely/strtree.py

```diff
@@ -34,317 +34,185 @@ class STRtree:
     at initialization. The tree is immutable and query-only, meaning that
     once created nodes cannot be added or removed.
 
-    An index is initialized from a sequence of geometry objects and
-    optionally a sequence of items. If items are not provided, the
-    indices of the geometry sequence will be used instead.
+    All operations return indices of the input geometries.  These indices
+    can be used to index into anything associated with the input geometries,
+    including the input geometries themselves, or custom items stored in
+    other object of the same length as the geometries.
 
-    Stored items and corresponding geometry objects can be spatially
-    queried using another geometric object.
+    Any mixture of geometry types may be stored in the tree.
 
     Parameters
     ----------
     geoms : sequence
         A sequence of geometry objects.
-    items : sequence, optional
-        A sequence of objects which typically serve as identifiers in an
-        application. This sequence must have the same length as geoms.
     node_capacity : int, default 10
         The maximum number of child nodes per parent node in the tree.
 
-    Examples
```

Comment:
I think these examples are better handled in the specific methods, so I removed them from the class level.